### PR TITLE
flakytest

### PR DESCRIPTION
**Pull request by infanta-deriv**
card - https://app.clickup.com/t/20696747/DPROD-3849 
Minor Fix for flaky test in Traders tool- Margin calculator in mobile testcase

### CODE REVIEW

File: cypress/e2e/full/tradersTool.cy.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ function marginCalculatorPage()
     calculator()
     cy.findByRole('img', {name: 'Plus'}).click()
     cy.get('[class^="accordion__AccordionHeaderItem"]').should('be.visible')
+    cy.wait(1000)
```
Comment (bilalmajid-deriv) [inline cypress/e2e/full/tradersTool.cy.js:32]: Is there no other way or no other element you can wait for instead of explicitly waiting

Comment (infanta-deriv) [reply cypress/e2e/full/tradersTool.cy.js:32]: Here when user clicks on "+" icon and soon clicks on "-" icon  , sometimes the content item is not minimized which is causing the test flaky . so only when added wait before clicking on "-" , the content can be properly minimized . So i added one sec wait . Or should i go with cy.findByRole('img', {name: 'Minus'}).should('be.visible') and then click it ? @bilalmajid-deriv

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Pull request merged**